### ISSUE TITLE
Separate functionality..

### ISSUE THREAD

**Issue by EC-O-DE** [idelairre/TumblrFox#3]
I've been looking at your code as I'd like to separate functionality...
Meaning I 'd want only the post filter options and not the Firebase thingy...
Could you help a bit?

**Comment by idelairre** [comment]
Hey, this is an old project but I’m down to help. Can you be more specific? In the chrome options menu you can disable Firebase and only use local storage. Is that what you mean?

**Comment by EC-O-DE** [comment]
Yes, but of course, I'd rather strip the package from all extra code. I've been going trhough the code - it requires much much focus - to separate in away it wont break.

**Comment by EC-O-DE** [comment]
Yes without the Firebase.. and~hey,, I didn't realize these these popovers require those massive amounts of attached code...

There must be a simpler way :hmm:
